Tools/i18n/pygettext.py

```diff
@@ -44,7 +44,7 @@
 
     -a
     --extract-all
-        Extract all strings.
+        Not implemented and will be removed in a future version.
```

Comment:
Maybe we can be more explicit?
```suggestion
        This option does not do anything and will be removed in a future version.
```